src/app/shared/libraries/accordion-item/accordion-item.component.html

```diff
@@ -0,0 +1,15 @@
+
+<div class="govuk-accordion__section">
+  <div class="govuk-accordion__section-header">
+    <h2 class="govuk-accordion__section-heading">
+      <span class="govuk-link govuk-!-font-size-36">{{title}}</span>
+      <span class="govuk-accordion__section-button" id="accordion-default-heading-{{title}}"></span>
+    </h2>
+  </div>
+  <div id="accordion-default-content" class="govuk-accordion__section-content">
```

Comment:
can we have a similar id as it is done on line 6, that is a sensible approach assuming all sections will be unique. 👍